### ISSUE TITLE
linux deploy trigger linux anti virus filesystem "Permission denied"

### ISSUE THREAD

**Issue by nicolas-f** [linuxdeploy/linuxdeploy#241]
I'm following the tutorial using QtQuickApp sample

But in this step I have an issue:

```sh
QtQuickApp/build$ ./linuxdeploy-x86_64.AppImage --appdir AppDir --output appimage 
linuxdeploy version 1-alpha (git commit ID 2d8eaf1), GitHub actions build 145 built on 2022-12-01 01:26:37 UTC
terminate called after throwing an instance of 'std::filesystem::__cxx11::filesystem_error'
  what():  filesystem error: status: Permission denied [/usr/bin/sentinelctl]
Abandon
```
sentinelctl is a linux antivirus that I can't disable or uninstall.

In verbose mode (-v0) there is a lot of file requests:

```
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/weave 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/apturl 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/canberra-gtk-play 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/ijs_pxljr 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/systemd 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/cpack 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/gcore 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/x86_64-linux-gnu-gcc-11 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/debconf-loadtemplate 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/chacl 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/pbmmask 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/git-receive-pack 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/gfortran-11 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/lzgrep 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/akonadi_maildir_resource 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/namei 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/ntfsrecover 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/qemu-storage-daemon 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/display-im6 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/tsort 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/mkdiskimage 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/qvkgen 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/gsnd 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/lzip-decompressor 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/pcre2-config 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/kioclient 
DEBUG: Doesn't match plugin regex, skipping: /usr/bin/kup-purger 
terminate called after throwing an instance of 'std::filesystem::__cxx11::filesystem_error'
  what():  filesystem error: status: Permission denied [/usr/bin/sentinelctl]
Abandon
```
Is there a command line option to handle the exception or to skip searching in folder /usr/ ?

Thanks

**Comment by nicolas-f** [comment]
Fixed, thanks for your support :)

**Comment by TheAssassin** [comment]
You fixed it yourself, you deserve my thanks! (Even though this was a kind of exotic problem... antivirus on _Linux_?)

**Comment by nicolas-f** [comment]
Lol yes, It is Research/University laptop. They want us to have an antivirus (useless?) and some "inventory" software installed on the system.

**Comment by TheAssassin** [comment]
Anyway. Binaries released. Have fun!

**Comment by olivierlemasle** [comment]
Hi, I had the exact same issue (with an old version of linuxdeploy, before this fix).
Actually, the issue is not the SentinelOne anti-virus is triggered.

It's simply that there's a symlink `/usr/bin/sentinelone` which is on the PATH, and which is owned by `root` with mode `0755`, which points to `/opt/sentinelone/bin/sentinelctl`, also owned by `root` but with mode `0700`.

linuxdeploy scans the PATH to search for plugins, follows the symlink and cannot read its target.

#191 and #227 are probably the same issue, which should now be fixed with #242.
